docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
+If you are an organization or a Project admin wanting to add a user to a project or an environment, the user has to create an account before they can contribute to the project.
 
 ## User roles
 
-Grant user permissions to the entire project:
-* **Project Viewer** - A project viewer can view all environments of the project.
-* **Project Administrator** - A project administrator can change settings, push code and execute actions on all environments of the project.
+| User role    | Description |
+| ------------ |-------------|
+|Project owner| There is only one and they can create multiple Project admins.  |
+|Project admin | Users who can configure project settings, manage other users,  administer environment permissions, push code, and execute actions on all project environments.|
 
-You can also define user access per environment type (Production, Staging, Development). Each permission level progressively increases access to the environments part of a given environment type:
+### Transfer a project ownership
 
-* **Viewer** - Viewer can view all environments of this type.
-* **Contributor** - Contributor can push code and branch all environments of this type.
-* **Administrator** - Administrator can change settings and execute actions on all environments of this type.
+As an organization owner, or as an organization user with the **manage plans** permission, you can transfer project ownership to a different organization, first invite that organization owner as a **Project admin** and then, submit a support ticket from your organization owner account to ask for the transfer. Transferring a project automatically transfers the upcoming subscription charges to the new owner. Once you transfer ownership, open a [support ticket](https://console.platform.sh/-/users/~/tickets) to request your account deletion.                                                                                                     
 
-{{< note theme="warning" title="Important" >}}
-After you add or remove a user from a project or an environment type, you will need to trigger a redeploy to propagate the access changes to each environment.
-You can redeploy by using the CLI command `platform redeploy` or the `Redeploy` button in the management console. 
-{{< /note >}}
+## Environment types
 
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
+Platform.sh offers three types of environment groups: Production, Staging, and
+Development. You can assign permissions to each user per environment so for each environment type, users always have the same set of permissions across all environments of the same type.
```

Comment:
The sentence here seems to be two ideas at the "so", so maybe breaking it with punctuation or otherwise might help. It might help to shorted the last part of it as well.

Also, I don't think you can assign permissions "per environment". I believe it's possible for each type, which is why the types were introduced. 

And a small point but the line break here doesn't seem ideally semantic.